domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -444,6 +451,45 @@ public static void closeAllClients() {
     CompositeFuture.join(list);
   }
 
+  static PgConnectOptions createPgConnectOptions(JsonObject sqlConfig) {
+    PgConnectOptions pgConnectOptions = new PgConnectOptions();
+    String host = sqlConfig.getString(HOST);
+    if (host != null) {
```

Comment:
I think != null is very convenient.. When we say that empty strings are "null", then what is an empty string.. Stripping blanks.. UNICODE blanks? I like that if a environment variable or JSON property is set in configuration, then that value is used. Whatever it is.

Reply:
It would be environment variable or JSON configuration error of course. I suppose it makes no difference of setting the host to `""` or not setting it at all. Would still get a connection error either way. Might not be worth validating and throwing exception without communication to database.